### PR TITLE
parallel_cam

### PR DESCRIPTION
**Pull request by PerryShafran-NOAA**
## Pull Request Testing ##

- [ ] Describe testing already performed for this Pull Request:</br>

These changes have been running in parallel in the emc.vpppg group account for awhile and works well.

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

Run the scripts as you normally would. Check the output to compare to your own runs. You may need to link to the /lfs/h2/emc/vpppg/noscrub/emc.vpppg/evs/v1.0 output if your runs need several days worth of data. You may recommend merging into the auth repo but you do not have the authority to perform the merge.

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [ ] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

At least 5 business days from receipt of this PR.

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

**Comment by PerryShafran-NOAA** [comment]
NOTE:  I have not yet incorporated the recent cam plots changes into this PR yet.  I need to test them first, and will be added to this PR once tested.

**Comment by AliciaBentley-NOAA** [comment]
Hi Perry,

A cursory look at this PR suggests that some version numbers were changed
from variables to hardcoded in the ecf scripts.
https://github.com/NOAA-EMC/EVS/pull/70/files

Can you please double check and potentially correct this before this is
tested by the developer? Thanks!

Take care,
Alicia
___________________________________
*Alicia M. Bentley, Ph.D.*
Physical Scientist | Model Evaluation Group
<https://www.emc.ncep.noaa.gov/users/meg/home/>
Global Model Verification Team Lead
NOAA/NWS/NCEP/EMC (VPPPG Branch)
Work Phone: ‪(301) 683-3783‬

**Comment by PerryShafran-NOAA** [comment]
Oops, yes, thanks for noting that.

Perry

**Comment by MarcelCaron-NOAA** [comment]
### PR Testing Summary
**Description**: I tested this parallel version of the CAM precip workflows (prep and stats) and the CAM grid2obs workflows (prep and stats).

**Results**: The precip workflows run without issue.  The grid2obs workflows each have one issue not related to your changes in this PR.  I ran the PR version of grid2obs with an added fix and it completed without issue.  [My fix is included in a current feature branch for CAM ptype verification](https://github.com/MarcelCaron-NOAA/EVS/commit/d9b066457ab54bee590823728acad36b2265a869) and for which a PR will be opened soon.  

**Conclusion**: _I approve this PR regarding the changes you’ve made to the CAM workflows_.  Regarding the fix, however, I suggest that this PR is either merged without the fix and we wait until the feature branch I mentioned is merged with develop, or I could submit the fix alone as a PR and that can be merged with the code in this PR before this PR is merged with develop.  

Let me know if you have any questions.

**Review by MarcelCaron-NOAA — APPROVED** [review]
### PR Testing Summary
**Description**: I tested this parallel version of the CAM precip workflows (prep and stats) and the CAM grid2obs workflows (prep and stats).

**Results**: The precip workflows run without issue.  The grid2obs workflows each have one issue not related to your changes in this PR.  I ran the PR version of grid2obs with an added fix and it completed without issue.  [My fix is included in a current feature branch for CAM ptype verification](https://github.com/MarcelCaron-NOAA/EVS/commit/d9b066457ab54bee590823728acad36b2265a869) and for which a PR will be opened soon.  

**Conclusion**: _I approve this PR regarding the changes you’ve made to the CAM workflows_.  Regarding the fix, however, I suggest that this PR is either merged without the fix and we wait until the feature branch I mentioned is merged with develop, or I could submit the fix alone as a PR and that can be merged with the code in this PR before this PR is merged with develop.  

Let me know if you have any questions.

**Comment by PerryShafran-NOAA** [comment]
Including @ShelleyMelchior-NOAA for comment.

I think we probably need to merge now.  It is possible that adding in the fix to this PR for the parallel may muddy things.  

Shelley - think we should merge this now, and then Marcel can submit his PR as a fix?  We need to approve his changes, but this PR is about approving the parallel changes.  

Perry

**Comment by ShelleyMelchior-NOAA** [comment]
I agree. I will merge this PR as is. Marcel can submit a follow on PR for the fix related to ptype.

File: ecf/cam/stats/jevs_hrrr_grid2obs_stats.ecf
Comment (ShelleyMelchior-NOAA) [inline ecf/cam/stats/jevs_hrrr_grid2obs_stats.ecf]: Change to VERF-DEV?

File: ecf/cam/stats/jevs_hrrr_precip_stats.ecf
Comment (ShelleyMelchior-NOAA) [inline ecf/cam/stats/jevs_hrrr_precip_stats.ecf]: Change to VERF-DEV?

**Review by ShelleyMelchior-NOAA — CHANGES_REQUESTED** [review]
Please check which PBS account should be used for job submissions: EVS-DEV or VERF-DEV.

**Comment by MarcelCaron-NOAA** [comment]
@ShelleyMelchior-NOAA I was able to use both.  I suppose the account used should be consistent among the ecf scripts ... is one preferred over the other?

**Comment by AliciaBentley-NOAA** [comment]
Hi Marcel and Shelley. Developers should exclusively use the VERF-DEV
project code for testing these PRs. Only the EVS parallel should be using
the EVS-DEV project code in order to accurately track EVS resources for an
estimate of ops. Developers should be approving the code with VERF-DEV. It
looks like the last few merged PRs used VERF-DEV, which is good. Mallory's
merged PR seems to still say EVS-DEV. We should clear that up. Thanks!
___________________________________
*Alicia M. Bentley, Ph.D.*
Physical Scientist | Model Evaluation Group
<https://www.emc.ncep.noaa.gov/users/meg/home/>
Global Model Verification Team Lead
NOAA/NWS/NCEP/EMC (VPPPG Branch)
Work Phone: ‪(301) 683-3783‬

File: ecf/cam/stats/jevs_hrrr_grid2obs_stats.ecf
Comment (MarcelCaron-NOAA) [reply ecf/cam/stats/jevs_hrrr_grid2obs_stats.ecf]: Yes please change this from "EVS-DEV" to "VERF-DEV".

File: ecf/cam/stats/jevs_hrrr_precip_stats.ecf
Comment (MarcelCaron-NOAA) [reply ecf/cam/stats/jevs_hrrr_precip_stats.ecf]: Yes please change this from "EVS-DEV" to "VERF-DEV"

**Comment by MarcelCaron-NOAA** [comment]
Thanks @AliciaBentley-NOAA.  In that case @PerryShafran-NOAA please change "EVS-DEV" in those two locations: 
```
ecf/cam/stats/jevs_hrrr_grid2obs_stats.ecf
```
```
ecf/cam/stats/jevs_hrrr_precip_stats.ecf
```
... to "VERF-DEV" before merging.  Thanks @ShelleyMelchior-NOAA for pointing that out.

File: ecf/cam/stats/jevs_hrrr_grid2obs_stats.ecf
Comment (PerryShafran-NOAA) [reply ecf/cam/stats/jevs_hrrr_grid2obs_stats.ecf]: Change is made.

File: ecf/cam/stats/jevs_hrrr_precip_stats.ecf
Comment (PerryShafran-NOAA) [reply ecf/cam/stats/jevs_hrrr_precip_stats.ecf]: Change is made.

**Review by ShelleyMelchior-NOAA — APPROVED** [review]
pbs account name modified correctly. Changes are approved. PR is approved for merge.

**Pull request merged**
